Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonOverviewViewModel.kt

```diff
@@ -149,4 +154,8 @@ class PersonOverviewViewModel @AssistedInject constructor(
 
     @AssistedFactory
     interface Factory : SimpleCWAViewModelFactory<PersonOverviewViewModel>
+
+    companion object {
+        private const val TAG = "PersonOverviewViewModel"
```

Comment:
consider to use something like TAG = someObject::class.java.simpleName
(optional)